.github/workflows/push-tests.yml

```diff
@@ -22,6 +22,12 @@ jobs:
   unit-test:
     runs-on: ubuntu-latest
     steps:
+    - name: Configure AWS Credentials
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-region: us-east-1
```

Comment:
Are we still using us-east-1? I thought we transitioned to using us-west-2

Reply:
We started using us-east-1 because the AWS Certificates Manager is in us-east-1

Reply:
We started using us-east-1 in order "to use an [ACM Certificate](https://docs.aws.amazon.com/acm/latest/userguide/acm-regions.html) with Amazon CloudFront". We can change back to us-west-2 but will need to keep the certificates in us-east-1. Keeping everything is a single regions seemed easier.